mutt_attach.c

```diff
@@ -671,8 +671,8 @@ int mutt_view_attachment(FILE *fp, struct Body *a, enum ViewAttachMode mode,
     pview.mode = PAGER_MODE_ATTACH;
     if (win->type == WT_INDEX)
     {
-      struct IndexSharedData *index_shared = dialog_find(win)->wdata;
-      index_shared->email = e;
+      // struct IndexSharedData *index_shared = dialog_find(win)->wdata;
```

Comment:
just delete these 2 lines, it was me doing something I didn't understand

Reply:
will do